.github/actions/cache-ui-dependencies/action.yaml

```diff
@@ -13,12 +13,15 @@ runs:
       with:
         path: |
           ui/deps
+          /github/home/.cache/yarn/v6
```

Comment:
How about adding a note to wherever v6 is picked to update this line when changing?
Or strip `/v6` here?

Reply:
I think we can drop version here so no need to update it. In theory we should obtain this path with `yarn cache dir` but then action need to have yarn installed